scripts/importing-Exporting-Config.js

```diff
@@ -0,0 +1,508 @@
+
+            var curr_Team_Num = curr_Team_Data_Obj_Key.split(' ').pop().split(':')[0]; //key is in format 'Team data for team ${teamNum}'
+            var curr_Team_Name = curr_Team_Data_Obj_Key.split(' ').pop().split(':')[1];
+            //console.log('currTeamName: ' + curr_Team_Name);
+            teams_In_Curr_Event[curr_Team_Num] = curr_Team_Name; //add to object for teams in every event
+            
+            for (var k = 0; k < curr_Team_Data_Obj_Value.length; k++)
+            {
+                var match_Or_Pit_Obj = curr_Team_Data_Obj_Value[k];
+                var match_Or_Pit_Obj_Key = Object.keys(match_Or_Pit_Obj)[0];
+                var match_Or_Pit_Obj_Value = match_Or_Pit_Obj[match_Or_Pit_Obj_Key]; //has array(s) (>1 if match because it has data for each match)
+
+                if (match_Or_Pit_Obj_Key.includes('Match Data'))
+                {
+                    //has an array full of elment/frequency pair objects in it for every match
+                    var num_Matches = match_Or_Pit_Obj_Value.length;
+
+                    for (var m = 0; m < num_Matches; m++)
+                    {
+                        var curr_Match_Arr = match_Or_Pit_Obj_Value[m];
+
+                        //need to find match num to set a local storage object, {'Match #' : number} is the first object in the array
+                        var match_Num_Obj = curr_Match_Arr[0];
+                        var match_Num_Key = Object.keys(match_Num_Obj)[0]; //only of length 1
+                        var match_Num = match_Num_Obj[match_Num_Key];
+
+                        var local_Storage_Obj_Name = matchAnswerObjName(season,curr_Event,curr_Team_Num,match_Num);
+                        var local_Storage_Obj_Value = JSON.stringify(curr_Match_Arr);
+                        localStorage.setItem(local_Storage_Obj_Name,local_Storage_Obj_Value);
+                    }
+                }
+
+                if (match_Or_Pit_Obj_Key.includes('Pit Data'))
+                {
+                    var local_Storage_Obj_Value = JSON.stringify(match_Or_Pit_Obj_Value); //is just an array with Q/A pair objects in it
+                    var local_Storage_Obj_Name = pitAnswerObjName(season,curr_Event,curr_Team_Num);
+                    localStorage.setItem(local_Storage_Obj_Name,local_Storage_Obj_Value);
+                }
+            }
+        }
+        //set cookie for teams in every event
+        var cookie_Name = event_data_cookie_name(season,curr_Event);
+        var cookie_Value = JSON.stringify(teams_In_Curr_Event);
+        setCookie(cookie_Name,cookie_Value,750);
+    }
+}
+
+function showImportModal(object)
+{
+    var keys = Object.keys(object);
+    var seasonList = object[keys[0]]; // season config list is the entry for the first key
+    
+    //add seasons to modal text
+    var modalText = 'This file contains data that will overwrite the following season(s): ';
```

Comment:
put a `<br/>` at the end of this line, maybe between each season name, and after them all.

Whitespace helps people to parse quickly. After the first time I've used your app, I expect to see a list of seasons, if I don't remember what's in this config file, I might want to pull this up and quickly skim the seasons in the file - which will be easier if they're not in the middle of a paragraph.